### PR TITLE
Allow approving PRs with missing size delta

### PR DESCRIPTION
**Pull request by danielrozenberg**
This will allow us to approve PRs that failed to get its bundle size delta calculated for any reason (e.g., missing GitHub webhook, botched `master` Travis build)

e.g., I couldn't approve this PR because the check.delta field was null https://github.com/ampproject/amphtml/pull/19703

### CODE REVIEW

File: bundle-size/app.js [hunk 1]
```diff
@@ -228,33 +228,41 @@ module.exports = app => {
...
     const owners = await getOwners(context.github);
+    const pullRequestId = context.payload.pull_request.number;
+    const headSha = context.payload.pull_request.head.sha;
 
     if (context.payload.review.state == 'approved' &&
         owners.includes(approver)) {
-      context.log(`Pull request ${context.payload.pull_request.number} ` +
-          'approved by a bundle-size keeper');
+      context.log(`Pull request ${pullRequestId} approved by a bundle-size ` +
+          'keeper');
```
Comment (rsimha) [inline bundle-size/app.js]: Right now, we have `bundle-size keeper` and `bundle-size squad`, correct?

Future task: Explain this terminology in the details page for the check.

Comment (danielrozenberg) [reply bundle-size/app.js]: Approval here means approving the PR itself, not specifically the check

Comment (danielrozenberg) [reply bundle-size/app.js]: Created an issue for the future task: #30

**Pull request merged**